sdk/reya_rest_api/client.py

```
@@ -704,6 +705,22 @@ async def get_spot_executions(self) -> SpotExecutionList:
+        """
+        Get spot trade busts (failed spot fills) for the owner wallet asynchronously.
+
+        Returns:
+            Spot trade busts
+
+        Raises:
+            ValueError: If no wallet address is available or API returns an error
+        """
+        wallet = self.owner_wallet_address
+        if not wallet:
+            raise ValueError("No wallet address available. Private key must be provided.")
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)